app/Distance.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace ConverterApp;
+
+class Distance extends Value
+{
+    public function __construct($value, $unit)
+    {
+        if ($value<0)
+       {
+     	    throw new \InvalidArgumentException("Please enter valid argument.");
```

Comment:
Each exception message should return meaningful information. Otherwise it will introduce unnecessary confusion to user of class.